src/main/kotlin/no/nav/yrkesskade/meldingmottak/pdf/domene/skademelding/PdfSkademelding.kt

```diff
@@ -1,18 +1,14 @@
-package no.nav.yrkesskade.meldingmottak.pdf.domene
-
-data class Soknadsfelt<T>(
-    val label: String,
-    val verdi: T
-)
+package no.nav.yrkesskade.meldingmottak.pdf.domene.skademelding
 
+import no.nav.yrkesskade.meldingmottak.pdf.domene.*
 
 data class PdfSkademelding(
     val innmelder: PdfInnmelder?,
     val skadelidt: PdfSkadelidt?,
     val skade: PdfSkade?,
     val hendelsesfakta: PdfHendelsesfakta?,
     val dokumentInfo: PdfDokumentInfo
-)
+) : PdfData()
```

Comment:
Denne var ny! Hva gjør denne? For min egen læring

Reply:
PdfData er tenkt som en forelderklasse for skademelding, skadeforklaring osv.  bl.a. PdfClient trenger dermed ikke flere implementasjoner.